src/store/actions.ts

```diff
@@ -77,6 +77,37 @@ export const unlockToken = (token: Token) => {
     };
 };
 
+export const lockToken = (token: Token) => {
+    return async (dispatch: any, getState: any) => {
+        const state = getState();
+        const ethAccount = getEthAccount(state);
+        const tokenBalances = getTokenBalances(state);
+
+        const contractWrappers = await getContractWrappers();
+        /* This should be 0, cero amount of tokens should be available if we lock them */
+        const numberOfUnlockedTokens = new BigNumber('0');
+
+        await contractWrappers.erc20Token.setProxyAllowanceAsync(
+            token.address,
+            ethAccount,
+            numberOfUnlockedTokens,
```

Comment:
I think it would be simpler to just inline the zero here. Even better, we could hide this behind some helper, but that's something we can do later.

Reply:
Done, now the zero is inline